src/test/java/teammates/test/cases/action/BaseActionTest.java

```diff
@@ -731,4 +731,8 @@ protected void verifyNumberOfEmailsSent(Action action, int emailCount) {
         assertEquals(emailCount, action.getEmailSender().getEmailsSent().size());
     }
 
+    protected String getPageResultDestination(String viewUri, boolean error, String userId) {
+        return viewUri + "?error=" + error + "&user=" + userId;
```

Comment:
You can use `Boolean.toString(error)` to use it with `addParamToUrl`, I would think having `error` as a boolean parameter is a better choice.